### PR TITLE
Migration-143-Migrate m UI menu bar

### PR DESCRIPTION
**Pull request by LindaNayeli104**
# What does this PR do?

Migrated the menu bar from using simple MUI components to a complex MUI component. Fix the functionality in light mode, changing the white color of the letters to black.

# Where should the reviewer start?

Run the project locally and review the menu bar in the home page, using dark and light theme.

# How should this be manually tested?

- Visual test comparing with the images attached. 
- Check the text colors using dark and light theme.
- Check that the bar is responsive.

# What are the relevant tickets?

#143

# Screenshots

**Before**

image

image

**After**

image

image

image

image

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Comment by jackbravo** [comment]
Could you provide screenshots also for tablet and mobile?

File: app/routes/projects/index.tsx [hunk 1]
```diff
@@ -14,22 +14,36 @@ import {
...
 import { SortInput } from "app/core/components/SortInput";
 import { searchProjects } from "~/models/project.server";
 import { requireProfile } from "~/session.server";
 import type { ProjectStatus } from "~/models/status.server";
 import { getProjectStatuses } from "~/models/status.server";
 import { ongoingStage, ideaStage } from "~/constants";
 import Link from "~/core/components/Link";
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
```
Comment (martinrobled0) [inline app/routes/projects/index.tsx:36]: is this variable being used?

**Review by leslye-patino — COMMENTED** [review]
Looks good, but please only review @martinrobled0 comments

File: app/routes/projects/index.tsx (hunk 1 above)
Comment (LindaNayeli104) [reply app/routes/projects/index.tsx:36]: Yes, I used it to make the component responsive. It helped me change the breakpoints values to match the old behavior. In line 262 is where I use that variable.

**Review by leslye-patino — COMMENTED** [review]
LGTM !

**Review by jackbravo — APPROVED** [review]
I didn't really did a through review. But saw the LGTM from @leslye-patino , but she forgot to approve. So marking approved so it can be merged.

**Pull request merged**